modules/pubsub/src/main/java/com/spotify/ffwd/pubsub/PubsubPluginSink.java

```diff
@@ -32,8 +32,8 @@
 import com.google.pubsub.v1.ProjectTopicName;
 import com.google.pubsub.v1.PubsubMessage;
 import com.spotify.ffwd.cache.WriteCache;
-import com.spotify.ffwd.model.Batch;
-import com.spotify.ffwd.model.Metric;
+import com.spotify.ffwd.model.v2.Batch;
+import com.spotify.ffwd.model.v2.Metric;
```

Comment:
This change is a bit sensitive. I'm curious how we could verify that Heroic consumer will be able to consume new metric serialized with updated Spotify100ProtoSerializer.java that uses v2 metric?